NEWS.md

```diff
@@ -28,6 +30,7 @@
 - Documentation of `friendly_type_of()` was provided (#22)
 - Minor updates to pull request review guidance were added (#201, #292)
 - Documentation of singular versus plural function argument names was added into the programming strategy vignette. Also documentation on the common arguments `missing_value` and `missing_values` was added. (#296)
+- Documentation highlighting the difference between `set_values_to` and `keep_source_vars` (#318)
```

Comment:
Should the "Programming Strategy" be updated?

Reply:
Weird, this commit isnt showing: https://github.com/pharmaverse/admiraldev/pull/323/commits/a2be991234f3f8d605ab7c2b4b53894f0e09ebc8

Reply:
worrisome - any ideas?

Reply:
Think it's actually fine, I was playing around with branch names/testing some funny-business, I just pushed the doc update again, its on the files changed now